scripts/xblock/README

```diff
@@ -0,0 +1,34 @@
+## xBlocks count script
+
+The script in this directory, xblock_counts.py when run will produce 2 csv files containing a summary 
+of the xBlock usage in courses.
+
+This script requires a list of xBlock types.  The xblock_studio_configuration.json file was the 
```

Comment:
Perhaps say that an example xblock_studio_configuration.json file is provided, as the one you have checked in certainly does not have all the available XBlocks listed.